code/headers/rhino.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+#include <base_module.hpp>
+#include <hwlib.hpp>
+#include <moving_platform.hpp>
+#include <qik_2s12v10.hpp>
+
+namespace r2d2::moving_platform {
+
+    class rhino_c : public moving_platform_c {
+    private:
+        qik_2s12v10_c qik_2s12v10_motorcontroller;
+        hwlib::adc &encode_m0;
+        hwlib::adc &encode_m1;
+        constexpr static uint8_t turn_factor = 66;
+        uint8_t turn_speed = 80;
+        constexpr static unsigned int adc_voltage = 3500;
+
+    public:
+        /**
+         * @brief
+         * Constructor rhino_c
+         * @param    qik_2s12v10_motorcontroller
+         * @param    comm
+         * @param    encode_m0
```

Comment:
A lot of params in this class have not been specified, please add some explanation to the variables. The beetle.hpp should give a good example for what they should look like

Reply:
0d7eec6ad3a37e4736af6680c0bcd19e4c1a6d4b

Reply:
https://github.com/R2D2-2019/moving_platform/pull/29/commits/2a61229eecbf8d3bba04bcb6c30a3b43a947b3a9

Reply:
> [0d7eec6](https://github.com/R2D2-2019/moving_platform/commit/0d7eec6ad3a37e4736af6680c0bcd19e4c1a6d4b)

Why was this one also linked here?

Reply:
error, misunderstanding of the comment

Reply:
That can always happen, the changes made in that hash are welcome since they were things I had originally missed.